splinterd/tests/admin/circuit_disband.rs

```diff
@@ -20,6 +20,7 @@ use splinterd::node::{Node, RestApiVariant};
 
 use crate::admin::circuit_commit::{commit_2_party_circuit, commit_3_party_circuit};
 use crate::admin::payload::{make_circuit_disband_payload, make_circuit_proposal_vote_payload};
+use crate::framework::circuit_builder::{CircuitData, ScabbardCircuitBuilderVeil as Scabbard};
```

Comment:
The rename here is unnecessarily confusing, suggest using ScabbardCircuitBuilderVeil directly.